schema/defs.yaml

```diff
@@ -287,12 +287,14 @@ description: Common schema definitions shared by all themes
         update_time: { "$ref": "#/$defs/propertyDefinitions/featureUpdateTime" }
         sources: { "$ref": "#/$defs/propertyDefinitions/sources" }
     namesContainer:
+      title: Names
       description: Properties defining the names of a feature.
       type: object
       properties:
         names:
           "$ref": "#/$defs/propertyDefinitions/allNames"
     levelContainer:
+      title: Level
```

Comment:
Could we adopt lowercase/snake_case to make these consistent with the current titles for the top-level schemas (*e.g.* `land_use`)?

Reply:
Fair enough — we'll have to change the title of all the containers to match